packages/tail-kit/src/components/modal/modal.tsx

```diff
@@ -101,15 +127,24 @@ export function Modal({
         data-testid="modal-overlay"
       >
         <div
-          className="flex flex-col w-full max-w-screen-sm max-h-full overflow-hidden bg-white rounded-md shadow-2xl"
+          className={`relative flex flex-col w-full max-w-screen-${maxWidth} max-h-full overflow-hidden bg-white rounded-md shadow-2xl`}
```

Comment:
So instead of writing something `max-w-screen-${maxWidth}` and adding purge styles use something like

```jsx
className={clsx('', (() => {
   switch(maxWidth) {
     case 'sm': 
       return 'max-w-screen-sm'
   }
})())}
```

as we would be removing the purge styles while doing theming.